### DISCUSSION TITLE
Add a config setting to report comments when in

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#411] [Ideas]
I asked in #370, why can't I report these comments, and it was because I was searching for a specific phrase. It's hard-coded that if you are in "Not Your Channel" mode that comments can only be reported if you are in Auto Smart Mode.

So maybe adding this as a config variable would be useful.

**Comment by ThioJoe** [comment]
It's basically to prevent abuse, like someone just searching "a" then spamming reports

**Reply by TechStudent10** [reply]
Oh yeah I didn't think of that. Isn't there a way to prevent this like if the length of the string is less or equal to 5 then don't do anything, or more spam prevention.
